backend/lib/endpoints/schema/geo.js

```diff
@@ -1,4 +1,11 @@
-// const geoSchema = Joi.object().keys({
-//   latitude: Joi.number().min(0).max(90).required(),
-//   longitude: Joi.number().min(-180).max(180).required(),
-// });
+const S = require("fluent-schema");
```

Comment:
Not really a fan of one letter variable names as it can make code less readable, although if this is the convention for this particular library then I am okay with keeping it this way. If it isn't convention, perhaps we can rename this to `fluentSchema`?

Reply:
@mannykary heh... yeah that's exactly what I thought
Although this is what I found in the Fastify documentation https://www.fastify.io/docs/master/Fluent-Schema/
I have usually always written full JSON schema documents in external files following the official standards, but the concept of "fluent API's" intends to make it possible to write these schema's much faster, I think that's also why they choose to use a single letter.

Personally I've learned to embrace short variable names in certain contexts such as a function that maps over a collection, since it can reduce cognitive overhead. And certain languages like Golang even consider this best practice: https://github.com/golang/go/wiki/CodeReviewComments#variable-names.

So I thought, if we use separate files for the schema's, and use this "fluent-schema" library, then why not follow the Fastify and fluent-schema documentation and use the short name?

Reply:
Fair enough, as I said I don't have strong opinions about it, especially if it's by convention. Lodash is another library that embraces that use of `_` 😄